open-craft/opencraft
# [BB-983] Cleanup stale load balancer configuration fragments

This PR adds code to clean up the stale, leaked HAProxy configuration fragments created by the integration tests , which were piling up and causing the Let's Encrypt renewals to hang and start hitting the rate limits. See Sven's comment on BB-140 for more details.

**Testing instructions:**
* Log in to the stage load balancer.
* Update the creation and modification time of some of the `integration-*` files to older than 3 days.
* Run the CI checks.
* Verify that the files were cleaned up and removed from the HAProxy configuration.
